src/Datadog.Trace.AspNet.Loader/Startup.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Datadog.Trace.ClrProfiler;
+
+namespace Datadog.Trace.AspNet.Loader
+{
+    /// <summary>
+    /// Temporary startup class to launch something from the GAC
+    /// </summary>
+    public static class Startup
+    {
+        private static bool httpModuleRegistered = false;
+
+        /// <summary>
+        /// Injects a call to load the AspNetModule
+        /// </summary>
+        [InterceptMethod(
+            Integration = "AspNet",
+            CallerAssembly = "System.Web",
+            CallerType = "System.Web.Compilation.BuildManager",
+            CallerMethod = "InvokePreStartInitMethodsCore",
```

Comment:
Was there a difference in intercepting at this point instead of in `InvokePreStartInitMethods`?

Reply:
Hmm, looking back at this decision, I don't think there's any difference between the two callsites. I think I ended up choosing this inner callsite based off a different solution I was testing but didn't end up using.